### PR TITLE
new communication protocol with server

### PR DESCRIPTION
**Pull request by PearTomas**
Fixed some issues discussed with Marek, but I can't recall what they were.

### CODE REVIEW

**Review by karkub — APPROVED** [review]
very good

**Comment by PearTomas** [comment]
[very good weather](https://youtu.be/8Xm0zLDyoVQ?t=4)

File: src/controller/V2.0.0/controllerV2/defines.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 //#define _DHCP set
 
 // simulation
-// #define SIMULATION
+ #define SIMULATION
```
Comment (pavelpetrovic) [inline src/controller/V2.0.0/controllerV2/defines.h:17]: huh?

**Review by pavelpetrovic — COMMENTED** [review]
we use v3 anyway.

File: src/controller/V2.0.0/controllerV2/defines.h (hunk 1 above)
Comment (PearTomas) [reply src/controller/V2.0.0/controllerV2/defines.h:17]: I'll fix it

**Pull request merged**